scripts/download_wikidata_oceans.py

```diff
@@ -0,0 +1,57 @@
+import requests
+from tqdm import tqdm
+import json
+import re
+
+url = 'https://query.wikidata.org/sparql'
+subquery = """
+SELECT ?item ?itemLabel ?altLabel
+WHERE
+{
+  ?item p:P31/ps:P31/wdt:P279* wd:Q9430.
+  ?article schema:about ?item.
+  OPTIONAL { ?item skos:altLabel ?altLabel . FILTER (lang(?altLabel) = "en") }
+  SERVICE wikibase:label { bd:serviceParam wikibase:language "[AUTO_LANGUAGE],en". }
+  FILTER (CONTAINS(STR(?item),
+"""
+combinations = [f"Q{n}" for n in range(10, 100)]
+waterbodies = {}
+for comb in tqdm(combinations):
```

Comment:
No need to download this data in batches as the total number of entries returned isn't large enough for the query to timeout.

Reply:
So line 17 to 19 are not needed since there is no possible timeout?

Reply:
The loop goes over different subsets of the possible query results. You need to remove the filter from the query and then run the query directly without any loop and work with the results. Try the query (after removing the filter) directly at https://query.wikidata.org/ and once you are satisfied with the query, extract information from it as done inside the loop in other scripts.